vector/src/main/res/values-fr/strings_tchap.xml

```diff
@@ -71,6 +71,9 @@
         <item quantity="other">Historique limité à %d jours</item>
     </plurals>
     <string name="tchap_room_invited_you">%s vous a invité</string>
+    <string name="tchap_fab_new_direct_message">Nouveau message direct</string>
+    <string name="tchap_fab_new_room_forum">Nouveau salon/forum</string>
```

Comment:
Please use String coming from Element "fab_menu_create_room" instead of creating a new one and update the value to "Nouveau salon/forum"